cdc/metadata.cc

```diff
@@ -86,33 +93,31 @@ cdc::stream_id cdc::metadata::get_stream(api::timestamp_type ts, dht::token tok)
         // Nothing protects us from that until we start using transactions for generation switching.
     }
 
-    auto it = gen_used_at(now);
+    auto it = gen_used_at(ts);
     if (it == _gens.end()) {
         throw std::runtime_error(format(
-                "cdc::metadata::get_stream: could not find any CDC stream (current time: {})."
-                " Are we in the middle of a cluster upgrade?", format_timestamp(now)));
+                "cdc::metadata::get_stream: could not find any CDC stream ({})."
+                " Are we in the middle of a cluster upgrade?", format_timestamp(ts)));
```

Comment:
It's hard to say what the printed timestamp signifies.
Maybe "could not find any CDC stream for timestamp {}"?

Reply:
done